project/iterative_sorting.py

```diff
@@ -1,36 +1,52 @@
 # Complete the selection_sort() function below in class with your instructor
-def selection_sort( arr ):
+def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
+        # (hint, can do in 3 loc)
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
         # TO-DO: swap
+        temp = arr[smallest_index]
+        arr[smallest_index] = arr[cur_index]
+        arr[cur_index] = temp
 
+    return arr
```

Comment:
Very good getting this one working. There's not much to say about this, which is good. 😄